### ISSUE TITLE
RubberBand2D interactor style

### ISSUE THREAD

**Issue by GuillaumeFavelier** [pyvista/pyvista#871]
It would nice to have the 'RubberBand2D' interactor style in PyVista to visualize/interact with 2D structures with the camera in parallel projection mode and the rotation disabled.

Example:

![Screenshot from 2020-08-03 11-41-55](https://user-images.githubusercontent.com/2365790/89200816-8d36dd00-d57e-11ea-83a4-9687a351b3d8.png)

Relevant to https://github.com/mne-tools/mne-python/pull/8082 and https://github.com/mne-tools/mne-python/pull/8125
